src/get_desc.py

```
@@ -475,15 +481,15 @@ async def get_bluray_section(self, meta, tracker):
-        tracker,
-        signature="",
-        comparison=False,
-        desc_header="",
-        image_list=None,
-        approved_image_hosts=None,
+        meta: dict[str, Any],
+        tracker: str,
+        signature: str = "",
+        comparison: bool = False,
+        desc_header: str = "",
+        image_list: list[dict[str, str]] = [],
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)